src/python/Utils/IteratorTools.py

```diff
@@ -31,7 +29,7 @@ def nestedDictUpdate(d, u):
     Code from Alex Matelli
     http://stackoverflow.com/questions/3232943/update-value-of-a-nested-dictionary-of-varying-depth
     """
-    for k, v in viewitems(u):
+    for k, v in u.items:
```

Comment:
@anpicci Andrea, I think this is the reason why unit tests are failing. This line should be 
```
    for k, v in u.items():
```

Can you please update this PR and give it another try?

Reply:
@amaltaro I have just committed the bug fix, sorry for the inconvenience!